code/mission/missionmessage.cpp

```diff
@@ -316,7 +316,8 @@ int add_avi( char *avi_name )
 	strcpy_s( extra.name, avi_name );
 	strcpy_s( extra.anim_data.filename, avi_name);
 	extra.num = -1;
-	generic_anim_load(&extra.anim_data);
+	generic_anim_load(&extra.anim_data);   // load only to validate the anim
+	generic_anim_unload(&extra.anim_data); // unload to not waste bmpman slots
```

Comment:
Doesn't this reset all the data in `extra.anim_data`? If that's the case, what's the reason to have those two load/unload calls in the first place if the data is not going to be used?